### PR TITLE
fix: shortcuts conflict with browser native shortcuts

### PR DESCRIPTION
**Pull request by majidzeno**
- Closes [#20222](https://github.com/cypress-io/cypress/issues/20222) 

### User facing changelog
Cypress shortcuts updated to not conflict with the browser native shortcuts

### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [ ] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

**Comment by majidzeno** [comment]
@jennifer-shehane Seems there is an issue with `desktop-gui` can you check 🙏  ?

**Comment by jennifer-shehane** [comment]
@majidzeno Is it possible to add a test around this behavior? The failures look like flaky tests.

**Comment by majidzeno** [comment]
@jennifer-shehane I tried to reproduce this issue locally but the test passed, I checked the test and the _`onMount`_ function seems to fire as expected, can we try to rerun the CI from the failed task once again? 🙏

**Comment by majidzeno** [comment]
@jennifer-shehane I rebased my branch to the `develop` and seems the issue is still happening, this is weird cause it had to be already fixed before [here](https://github.com/cypress-io/cypress/pull/18979), I will check this PR to find the why it is still `flaky`

**Comment by majidzeno** [comment]
@jennifer-shehane the flaky test is `this.ipc.getUserEditor` is not triggered every time, I think the reason behind that is the nature of `useEffect` which is used in `onMount`, it is triggered in first but the events will not flush until the next update, I  think this solution is a bit hacky but I forced the component to update by changing the stat, check the last commit.

File: packages/reporter/cypress/integration/shortcuts_spec.ts [hunk 1]
```diff
@@ -135,5 +135,15 @@ describe('shortcuts', function () {
       cy.get('button.stop').trigger('mouseover')
       cy.get('.cy-tooltip').should('have.text', 'Stop Running S')
     })
+
```
Comment (BlueWinds) [inline packages/reporter/cypress/integration/shortcuts_spec.ts:138]: @majidzeno - I went ahead and pushed a test for these changes onto your branch, this is what @jennifer-shehane was asking for.

Looks good to me. :+1:

**Review by emilyrohrbough — APPROVED** [review]

**Comment by jennifer-shehane** [comment]
Kitchensink is failing in 'test-cypress-scaffold' due to known reasons that we're working on resolving.

**Comment by BlueWinds** [comment]
Given that we know the test failure is unrelated (and I'm working on it in a separate ticket), I'm going to merge this as-is. Thanks for your contribution, @majidzeno :)

**Comment by majidzeno** [comment]
@BlueWinds @jennifer-shehane Thanks alot 🙏 
That was my first contribution to cypress btw 😄

**Pull request merged**